### DISCUSSION TITLE
Upgrading from 0.7 -> 0.16.5 fails

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#2711] [Q&A] [answered]
`Failed to apply migration 1673167670_multi_match_migrate.go: SQL logic error: no such column: indexes (1)`

Do updates need to be applied by running intermediate versions of PocketBase?

**Comment by DanyWallace** [comment]
I think an incremental upgrade is the way to go here due how much changes.

**Comment by ganigeorgiev** [comment, marked as answer]
v0.8+ comes with some breaking changes due to the `users` field and `profiles` collection merge and if you use v0.7.x it cannot be updated by just replacing the binary (note that it may require changes in the client-side code). 

To migrate, from v0.7.x, you can check https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x. You should be able to follow this guide with the v0.13.4 binary (before the collections indexes).
After upgrading to v0.13.4, you can try replacing the binary with the latest one. For more details, please review the [CHANGELOG](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md).
